vizro-ai/docs/pages/explanation/why-use-vizro-ai.md

```diff
@@ -0,0 +1,12 @@
+# Why use Vizro-AI over ChatGPT?
+
+Vizro-AI offers an easy way to turn natural language instructions into professional visualizations, like interactive charts and dashboards, without needing advanced coding skills.
+
+Like ChatGPT, Vizro-AI uses generative AI to understand and execute your instructions in English, or any other language. However, there are few advantages of using Vizro-AI over ChatGPT.
+
+- **Tailored for data visualization**: Vizro-AI is specifically designed to create and design charts and dashboards.
```

Comment:
```suggestion
- **Tailored for data visualization**: Vizro-AI is specifically designed to create charts and dashboards with in-built beautiful visual design best practices.
```

I would rephrase to something like the above because Vizro-AI and Vizro overall don't assist in designing new charts and dashboards. Instead, they provide an opinionated design through Vizro, but they don't "help" in the sense of creating new designs, which I would partially understand from above